clarity/src/vm/analysis/type_checker/tests/mod.rs

```diff
@@ -74,6 +75,92 @@ fn ascii_type(size: u32) -> TypeSignature {
     TypeSignature::SequenceType(StringType(ASCII(size.try_into().unwrap()))).into()
 }
 
+#[test]
+fn test_to_consensus_buff() {
```

Comment:
Do you want to add a test for standard and contract principals?

Reply:
Added in d8f7628